recipes/uchardet/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
```suggestion
conan_basic_setup(TARGETS)
```

Reply:
not sure why but with `TARGETS` it didn't work. now that I see it this seems quite odd

Reply:
See https://github.com/conan-io/conan-center-index/pull/8036/files#r749606389.

By using `TARGETS`, the include and library directories of dependencies are not added.
You used `uchardet` there, which is the name of the library file, not the target name.
When using targets, you need to let go of filenames of libraries, and only link to targets.

Reply:
cool - thank you for the explanation!